media/transcoder/create_job_with_standalone_captions.go

```diff
@@ -91,15 +96,32 @@ func createJobWithStandaloneCaptions(w io.Writer, projectID string, location str
 							},
 						},
 						{
-							Key: "vtt-stream0",
+							Key: "vtt_stream_en",
 							ElementaryStream: &transcoderpb.ElementaryStream_TextStream{
 								TextStream: &transcoderpb.TextStream{
-									Codec: "webvtt",
+									Codec:        "webvtt",
+									LanguageCode: "en-US",
+									DisplayName:  "English",
 									Mapping: []*transcoderpb.TextStream_TextMapping{
 										{
-											AtomKey:    "atom0",
-											InputKey:   "caption_input0",
-											InputTrack: 0,
+											AtomKey:  "atom0",
+											InputKey: "subtitle_input_en",
+										},
+									},
+								},
+							},
+						},
+						{
+							Key: "vtt_stream_es",
```

Comment:
quibble: the level of indentation here makes the sample less readable. I suggest breaking some of these objects out and instantiating them separately outside of the slice.

Reply:
refactored